class.jetpack-idc.php

```diff
@@ -139,10 +168,17 @@ function enqueue_idc_notice_files() {
 			)
 		);
 
+		wp_register_style(
+			'jetpack-dops-style',
+			plugins_url( '_inc/build/admin.dops-style.css', JETPACK__PLUGIN_FILE ),
+			array(),
+			JETPACK__VERSION
+		);
+
```

Comment:
If everything else is OK with this PR, it'd help if we moved forward with merging this and created an issue for handling getting the `.dops-button` styles.